client/main.lua

```diff
@@ -100,22 +98,22 @@ local function setVehicleDoorLock(veh, state, anim)
                     TaskPlayAnim(cache.ped, 'anim@mp_player_intmenu@key_fob@', 'fob_click', 3.0, 3.0, -1, 49, 0, false, false, false)
                 end
 
-                TriggerServerEvent("InteractSound_SV:PlayWithinDistance", 5, "lock", 0.3)
-
+                TriggerServerEvent('InteractSound_SV:PlayWithinDistance', 5, 'lock', 0.3)
                 NetworkRequestControlOfEntity(veh)
                 if state then
-                    state = state == true and 2 or 1
+                    state = not state and 1 or 2
```

Comment:
I think in this case the positive expression is more readable than the negation. Curious what this logic is trying to express though as the if condition ensures state is not false and not nil, so curious when state is explicitly true versus when it's a number.

Reply:
Well, it should only expect a boolean or a nil value, as that's what the case is for. And the explicit true is just a matter of old practices from QB times

Reply:
I will flip it though because you're right, a positive expression looks better